### PR TITLE
NEWS.md: add more new 2.60 features

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

**Review by alfonsosanchezbeato — APPROVED** [review]
LGTM

File: NEWS.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 * Apparmor userspace is vendored inside the snapd snap
 * Added a default-configure hook that exposes gadget default configuration
   options to snaps during first install before services are started
+* Allow install from initrd to speed up the initial installation
```
Comment (pedronis) [inline NEWS.md:6]: shouldn't this say more about current limitations?

**Review by pedronis — APPROVED** [review]

File: NEWS.md (hunk 1 above)
Comment (mvo5) [reply NEWS.md]: Good point, I added a note on the (current) limitations and added also a few more of the 2.60 featues

**Review by MiguelPires — APPROVED** [review]

File: NEWS.md [hunk 2]
```diff
@@ -3,6 +3,13 @@
 * Apparmor userspace is vendored inside the snapd snap
 * Added a default-configure hook that exposes gadget default configuration
   options to snaps during first install before services are started
+* Allow install from initrd to speed up the initial installation for
+  systems that do not have a install-device hook
+* New `snap sign --chain` flag that append the account and account-key
+  assertions
+* Support validation-sets in the model assertion
+* Support new "min-size" field in gadet.yaml
```
Comment (pedronis) [inline NEWS.md:11]: typo: gadget

**Review by alfonsosanchezbeato — APPROVED** [review]

File: NEWS.md [hunk 3]
```diff
@@ -3,6 +3,13 @@
 * Apparmor userspace is vendored inside the snapd snap
 * Added a default-configure hook that exposes gadget default configuration
   options to snaps during first install before services are started
+* Allow install from initrd to speed up the initial installation for
+  systems that do not have a install-device hook
+* New `snap sign --chain` flag that append the account and account-key
```
Comment (degville) [inline NEWS.md:8]:
```suggestion
* New `snap sign --chain` flag that appends the account and account-key
```

**Pull request merged**